appcelerator-archive/amp
# Adding dep to the gotools image

https://techweb.axway.com/jira/browse/AMP-131

This PR adds `dep` to the `appcelerator/gotools` image in order to install vendor without adherence to the environment.

## How to test
```
$ ampmake clean-vendor install-vendor
```
Make sure there are no differences with the branch.

```
$ ampmake clean build
```
Make sure the build passes.
